server/profile/profile.go

```diff
@@ -22,6 +24,22 @@ type Profile struct {
 	DSN string `json:"-"`
 	// Version is the current version of server
 	Version string `json:"version"`
+	// Feat is the feature set of server, split by comma
+	Feat string `json:"feature"`
+}
+
+const (
+	FeatureSSO       Feature = "SSO"
+	FeatureStorageS3 Feature = "STORAGE_S3"
+)
```

Comment:
I think you are not using these two enum values.

Reply:
these two is using in frontend, holding here for future using in backend